src/python/tensorflow_cloud/core/tests/unit/deploy_test.py

```diff
@@ -186,7 +186,6 @@ def test_request_dict_without_user_args(self, MockDiscovery):
 
     @patch("tensorflow_cloud.core.deploy.discovery")
     def DISABLED_test_request_dict_with_TPU_worker(self, MockDiscovery):
-        # TODO(psv): Fix broken test.
```

Comment:
If we are removing this then DISABLED should be removed from the test name as well.

Reply:
got it, enabled the tests. Since the TPU version is now verified in `validate.py` I changed the logic in this test to pick up the TPU version from environment instead of hardcoding to 2.1.